packages/openapi3/src/openapi.ts

```diff
@@ -328,6 +329,12 @@ function createOAPIEmitter(program: Program, options: ResolvedOpenAPI3EmitterOpt
     if (!serviceNs) {
       return;
     }
+    const commonProjections: ProjectionApplication[] = [
+      {
```

Comment:
feels like we need a few tests using projected json name

Reply:
yes, this PR wasn't ready. I think it really needs the `runProjection` change #861. Not that much for the openapi3 emitter as it runs the json all the time but for the cadl-autorest and the controller we'd want to run multiple decorators.